dll/win32/shell32/shellmenu/CMenuBand.cpp

```diff
@@ -930,8 +930,18 @@ HRESULT  CMenuBand::_KeyboardItemChange(DWORD change)
             else
                 tb = m_staticToolbar;
         }
+        else
```

Comment:
This else-block looks like the VK_UP || VK_END case from above. Is it to be expected?
If so, would it be better to move the middle-block VK_DOWN || VK_HOME first, and make the VK_UP || VK_END the default?

Maybe changing this to a switch could be better?
```c
        switch (change)
        {
        case VK_DOWN: case VK_HOME:
            if (m_SFToolbar)
                tb = m_SFToolbar;
            else
                tb = m_static Toolbar;
            break;

        case VK_UP: case VK_END:
        default:
            if (m_staticToolbar)
                tb = m_staticToolbar;
            else
                tb = m_SFToolbar;
            break;
        }
```
----
I personally wonder why the if-tests for the toolbars are done the way they are: why sometimes it's the `m_ staticToolbar` that is being checked, and other times it's `m_SFToolbar` instead.